.github/workflows/coverage-report.yml

```diff
@@ -0,0 +1,56 @@
+name: Coverage report
+
+on:
+  workflow_run:
+    workflows: ["tests"]
```

Comment:
Could you add a comment explaining that this makes the workflow here trigger after every completed run of the `tests` workflow?

Reply:
Done.